docs/recommended_tools.md

```diff
@@ -17,6 +17,8 @@ For an overview over which icons are available, you can install the [Icon Librar
 
 For using custom icons and icon-sets, you may want to look into [Symbolic Preview](https://flathub.org/apps/details/org.gnome.design.SymbolicPreview).
 
+Freedesktop.org also provides a [specification for standard icon-names](https://specifications.freedesktop.org/icon-naming-spec/icon-naming-spec-latest.html#names) that Gnome uses.
```

Comment:
On the eternal search for "Does GTK/Gnome have any kind of official list of icons you can use?" (which I still haven't found a definitive list for), I stumbled over this link and thought it'd be useful enough to include.

Reply:
I usually use the `IconLibrary` application mentioned above, but I would be happy to also include this link.